TrafficCapture/dockerSolution/src/main/docker/migrationConsole/Dockerfile

```diff
@@ -128,8 +129,24 @@ RUN echo '. /etc/profile.d/bash_completion.sh' >> ~/.bashrc && \
     echo 'workflow util completions bash > /usr/share/bash-completion/completions/workflow' >> ~/.bashrc && \
     echo 'PS1="(\t) \[\e[92m\]migration-console \[\e[0m\](\w) -> "' >> ~/.bashrc
 
+# Install YQ
```

Comment:
nit: python can do yq transforms pretty well too.  yq isn't as ubiquitous as jq or python.

Reply:
What do you need yq for?